tedana/reporting/html_report.py

```diff
@@ -138,44 +143,87 @@ def _update_template_bokeh(bokeh_id, info_table, about, prefix, references, boke
     -------
     HTMLReport : an instance of a populated HTML report
     """
-    resource_path = Path(__file__).resolve().parent.joinpath("data", "html")
-
     # Initial carpet plot (default one)
     initial_carpet = f"./figures/{prefix}carpet_optcom.svg"
 
+    # Get the figures directory
+    figures_dir = "figures"
+
+    # List all files in the figures directory
+    files_in_figures = os.listdir(figures_dir)
```

Comment:
Any chance the cwd isn't the output directory for the report? If so, then relative paths in the HTML report would be fine, but this line will fail.

Reply:
```suggestion
    files_in_figures = os.listdir(os.path.abspath(figures_dir))
```

Reply:
If I'm right, then this will still fail, but it'll at least be clear that the location of "figures_dir" isn't where we'd expect it because the cwd isn't the output directory.

Reply:
I was right!

> FAILED tedana/tests/test_integration.py::test_integration_four_echo - FileNotFoundError: [Errno 2] No such file or directory: '/tmp/src/tedana/figures'

Reply:
It should be `outputs/figures`